packages/ui/src/theme/styles/overrides/home.css

```
@@ -87,11 +96,74 @@ html .rp-home-hero__actions {
+
+@media (max-width: 1184px) {
+  .zp-feature-section {
+    padding-left: 16px;
+    padding-right: 16px;
+  }
+}
+
+/* ── Feature grid ───────────────────────────────────────────────── */
+.zp-feature-grid {
+  --feature-grid-gap: 12px;
+  display: flex;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)